pkg/model/github_context.go

```diff
@@ -169,7 +169,9 @@ func (ghc *GithubContext) SetRepositoryAndOwner(ctx context.Context, githubInsta
 	if ghc.Repository == "" {
 		repo, err := git.FindGithubRepo(ctx, repoPath, githubInstance, remoteName)
 		if err != nil {
-			common.Logger(ctx).Warningf("unable to get git repo (githubInstance: %v; remoteName: %v, repoPath: %v): %v", githubInstance, remoteName, repoPath, err)
+			// nektos/act is used as a default action, so why not a repo?
```

Comment:
Great idea!

Reply:
The main intent with this PR is simply to clean the output so it's cleaner for a blog post I'm making tomorrow.

I'm translating code for Chapter 8 of Bootstrapping Microservices from Node to Go.  The chapter uses GitHub Actions for CI/CD and the purpose of the series is to keep everything local, thus my interest in `act`.

It would be great if this could be merged by then.

Reply:
> I have technically a conflict of interest in promoting act as standalone cli tool for GitHub Actions, since I'm developing a far less known similar tool that does a very similar thing with the official actions/runner.

Fortunately, running minikube via act was a pain, so that idea's been abandoned until I can get a few new PRs made.

Are you able to share the name of your lesser known tool?